statsmodels/discrete/tests/test_sandwich_cov.py

```diff
@@ -614,6 +631,24 @@ def test_cov_options(self):
         assert_allclose(res1a.bse, bse, rtol=1e-6)
 
 
+
+class TestGLMGaussHACUniform2(TestGLMGaussHACUniform):
+
+    @classmethod
+    def setup_class(cls):
+
+        cls.cov_type = 'HAC'
+
+        kwds={'kernel':sw.weights_uniform, 'maxlags':2}
+        mod1 = GLM(endog, exog, family=families.Gaussian())
+        cls.res1 = mod1.fit(cov_type='HAC', cov_kwds=kwds)
```

Comment:
If this became `cov_type=cls.cov_type` the `setup_class` methods in this module could become shared/parametrized.